### ISSUE TITLE
xmp profile on H670 chipset does not seem to be detected

### ISSUE THREAD

**Issue by arigit** [anrieff/libcpuid#163]
Running the lastest cpu-x release (from github), I see:

![image](https://user-images.githubusercontent.com/652673/173206153-f41fbfc8-52ad-45b4-af60-427f72e4f438.png)

In BIOS, XMP2.0 profile 1 is selected, which is 4266Mhz. 
dmidecode shows this:
```Memory Device
	Array Handle: 0x000B
	Error Information Handle: Not Provided
	Total Width: 64 bits
	Data Width: 64 bits
	Size: 16 GB
	Form Factor: DIMM
	Set: None
	Locator: Controller0-ChannelA-DIMM0
	Bank Locator: BANK 0
	Type: DDR4
	Type Detail: Synchronous
	Speed: 2400 MT/s
	Manufacturer: Kingston
	Serial Number: xxxxxxxxxxxxx
	Asset Tag: xxxxxxxxxxxxxxxxxx
	Part Number: KF4266C19D4/16GX    
	Rank: 2
	Configured Memory Speed: 4267 MT/s
	Minimum Voltage: 1.2 V
	Maximum Voltage: 1.4 V
	Configured Voltage: 1.2 V
	Memory Technology: DRAM
	Memory Operating Mode Capability: Volatile memory
	Firmware Version: Not Specified
	Module Manufacturer ID: Bank 2, Hex 0x98
	Module Product ID: Unknown
	Memory Subsystem Controller Manufacturer ID: Unknown
	Memory Subsystem Controller Product ID: Unknown
	Non-Volatile Size: None
	Volatile Size: 16 GB
	Cache Size: None
	Logical Size: None
```

The entry above for "configured memory speed" seems to confirm that the actual speed is indeed 4266. 
Also testing with passmark with and without the xmp profile applied shows a large difference in performance so it seems that indeed the profile is applied.

This is on a h670 chipset motherboard.

**Comment by X0rg** [comment]
Not related to libcpuid: memory information in CPU-X are provided by dmidecode.

**Comment by X0rg** [comment]
@arigit fixed by X0rg/CPU-X@5b9c2ea5b5cd0c46554488af36d15a284a0c230c.
